### PR TITLE
Translate docs/docs/recipes/working-with-starters.md

### PR DESCRIPTION
**Pull request by k0mpreni**

### CODE REVIEW

File: docs/docs/recipes/working-with-starters.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
 ---
-title: "Recipes: Working with Starters"
+title: "Recette: Travailler avec des configurations de démarrage"
 tableOfContentsDepth: 1
 ---
 
-[Starters](/docs/starters/) are boilerplate Gatsby sites maintained officially, or by the community.
+Les [Configurations de démarrage](/docs/starters/) sont des sites Gatsby passe-partout entretenus officiellement ou par la communauté.
```
Comment (SamuelAlev) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
Les [configurations de démarrage](/docs/starters/) sont des sites Gatsby passe-partout maintenus officiellement ou par la communauté.
```

File: docs/docs/recipes/working-with-starters.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-2. Generate a new site based on the starter. In the terminal, run:
+2. Générez un nouveau site basé sur la configuration de démarrage. Dans le terminal, exécutez:
 
 ```shell
-gatsby new {your-project-name} {link-to-starter}
+gatsby new {le-nom-de-votre-projet} {le-lien-de-la-configuration-de-demarrage}
 ```
 
-> _Don't run the above command as-is -- remember to replace {your-project-name} and {link-to-starter}!_
+> _Ne pas exécuter la commande ci-dessus telle quelle -- n'oubliez pas de remplacer {le-nom-de-votre-projet} et {le-lien-de-la-configuration-de-demarrage}!_
```
Comment (SamuelAlev) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
> _Ne pas exécuter la commande ci-dessus tel quel -- n'oubliez pas de remplacer {le-nom-de-votre-projet} et {le-lien-de-la-configuration-de-demarrage}!_
```

File: docs/docs/recipes/working-with-starters.md [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
 
-3. Run your new site:
+3. Exécutez votre nouveau site:
```
Comment (SamuelAlev) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
3. Lancez votre nouveau site:
```

**Review by SamuelAlev — DISMISSED** [review]

File: docs/docs/recipes/working-with-starters.md [hunk 4]
```diff
@@ -1,38 +1,38 @@
 ---
-title: "Recipes: Working with Starters"
+title: "Recette: Travailler avec des configurations de démarrage"
```
Comment (mindsers) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
title: "Recettes : Travailler avec des configurations de démarrage"
```

File: docs/docs/recipes/working-with-starters.md [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
 ```shell
-gatsby new {your-project-name} {link-to-starter}
+gatsby new {le-nom-de-votre-projet} {le-lien-de-la-configuration-de-demarrage}
 ```
 
-> _Don't run the above command as-is -- remember to replace {your-project-name} and {link-to-starter}!_
+> _Ne pas exécuter la commande ci-dessus tel quel -- n'oubliez pas de remplacer {le-nom-de-votre-projet} et {le-lien-de-la-configuration-de-demarrage}!_
 
-3. Run your new site:
+3. Lancez votre nouveau site:
```
Comment (mindsers) [inline docs/docs/recipes/working-with-starters.md]:
```suggestion
3. Lancez votre nouveau site :
```

**Review by mindsers — CHANGES_REQUESTED** [review]
We're almost there! Here are some needed changes.

**Review by mindsers — APPROVED** [review]

File: docs/docs/recipes/working-with-starters.md (hunk 3 above)
Comment (k0mpreni) [reply docs/docs/recipes/working-with-starters.md]: Is it good for you @SamuelAlev ? I would love to start on a new file :)

**Pull request merged**